tests/Cookie/CookieJarTest.php

```diff
@@ -366,7 +369,7 @@ public function testCanConvertToAndLoadFromArray()
     public function testAddsCookiesWithEmptyPathFromResponse()
     {
         $response = new Response(200, array(
-            'Set-Cookie' => "fpc=foobar; expires=Fri, 02-Mar-2019 02:17:40 GMT; path=;"
+            'Set-Cookie' => "fpc=foobar; expires=Fri, {$this->futureExpirationDate()}; path=;"
```

Comment:
Though not critical for strtotime(), day `Fri, ` is left?

Reply:
@ranvis Thanks, nice spot, it was sloppy coding on my part, fixed now.